app/default/account_routes.py

```diff
@@ -146,11 +150,10 @@ def dashboard():
         # find and display applications with this
         # fund and round else return 404
         template_name = "dashboard_single_fund.html"
-        round_details = get_round_data_by_short_names(
-            fund_short_name,
-            round_short_name,
+        fund_details, round_details = get_fund_and_round(
+            fund_short_name=fund_short_name, round_short_name=round_short_name
```

Comment:
For consistency, these should probably be separated onto their own lines, as [seen below](https://github.com/communitiesuk/funding-service-design-frontend/pull/366/files#diff-25ffeea551d125947892e07d498380fe030e5f7c005ba15e165622de5d567f0eR242)